payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -88,7 +89,11 @@ internal abstract class BaseAddPaymentMethodFragment(
                 formFragment.paramMapLiveData.observe(viewLifecycleOwner) { paramMap ->
                     sheetViewModel.updateSelection(
                         paramMap?.run {
-                            selectedPaymentMethod.paymentMethodCreateParams(this)
+                            PaymentMethodCreateParams.createWithOverriddenParamMap(
+                                PaymentMethodCreateParams.Type.GenericPaymentMethod(paramMap["type"] as String),
+                                paramMap,
+                                setOf("PaymentSheet")
+                            )
```

Comment:
I think we can directly call the constructor instead of using this static method.

Reply:
Done